utils/docker/examples/daos-base/el8/Dockerfile

```diff
@@ -64,25 +64,23 @@ RUN	if [ -n "$BUST_CACHE" ] ; then
 	done &&                                                                                    \
 	echo "[INFO] Installing DAOS" ;                                                            \
 	dnf install                                                                                \
-		daos-${DAOS_VERSION}                                                               \
-		daos-client-${DAOS_VERSION}                                                        \
-		daos-client-tests-${DAOS_VERSION} &&                                               \
+		sudo                                                                               \
+		daos-${DAOS_VERSION} &&                                                            \
 	dnf clean all
 
 # Install DAOS client user and group information
-ARG	DAOS_CLIENT_UNAME=""
-ARG	DAOS_CLIENT_UID=""
-ARG	DAOS_CLIENT_GNAME=""
-ARG	DAOS_CLIENT_GID=""
-RUN	for it in DAOS_CLIENT_UNAME DAOS_CLIENT_UID DAOS_CLIENT_GNAME DAOS_CLIENT_GID ; do         \
-		if eval "[[ -z \$$it ]]" ; then                                                    \
-			echo "[ERROR] Docker build argument $it is not defined" ;                  \
-			exit 1 ;                                                                   \
-		fi ;                                                                               \
-	done ;                                                                                     \
-	echo "[INFO] Adding DAOS client user ${DAOS_CLIENT_UNAME}" ;                               \
-	groupadd -g ${DAOS_CLIENT_GID} ${DAOS_CLIENT_GNAME} &&                                     \
-	useradd -g ${DAOS_CLIENT_GID} -u ${DAOS_CLIENT_UID} ${DAOS_CLIENT_UNAME}
+ARG	DAOS_CLIENT_UNAME="root"
+ARG	DAOS_CLIENT_UID="0"
+ARG	DAOS_CLIENT_GNAME="root"
+ARG	DAOS_CLIENT_GID="0"
+RUN	if ! getent group ${DAOS_CLIENT_GNAME} > /dev/null 2>&1; then                              \
```

Comment:
This should run with 'set -e', there's a chance the groups could conflict as we saw with the change from sles 15.4 to 15.5 and from reading this code I'm not sure a failure of useradd here would propagate to  a failure of the build/compose.

Reply:
I was expecting that set -e is already set in a dockerfile.  If not I will add it.

Reply:
I don't think there is a way to set it for the dockerfile, each RUN statement is it's own shell script so you need to set if for each one or use `&&` notation between each command.

Reply:
I will do both.
Normally, I am using `&&`  when it is needed (not needed from my point of view with echo stuff).
Anyways, I will check if I have not miss some and I will add the `set -e -o pipefail` in the RUN statement.